esrally/telemetry.py

```diff
@@ -2225,3 +2226,60 @@ def record(self):
         }
 
         self.metrics_store.put_doc(doc, level=MetaInfoScope.cluster)
+
+
+class FieldDiskUsage(TelemetryDevice):
```

Comment:
Yeah, I guess it'd be `DiskUsage` to line it up. I didn't know the name of the API when I wrote this and never bothered to rename it because, honestly, I don't like the current API name. But I'm happy to rename it to line up.